aviary/subsystems/subsystem_builder_base.py

```diff
@@ -319,22 +318,22 @@ def preprocess_inputs(self, aviary_inputs):
         """
         return aviary_inputs
 
-    def get_outputs(self):
+    def get_timeseries(self):
```

Comment:
For this rename, I recommend adding `get_outputs` back, and give it a deprecation warning (or even an error). This will help us catch them.